blogposts/announcing-sourcegraph-3.0.md

```diff
@@ -0,0 +1,243 @@
+- [Deploy to more kinds of clusters](#deploy-to-more-kinds-of-clusters): if our existing 1-Docker-container or Kubernetes deployment schemes don't work for you.
+- [Authentication and repository permissions from GitHub](#authentication-and-repository-permissions-from-github): allow user sign-in and enforce repository access permissions from GitHub or GitHub Enterprise.
+- [Nginx-based HTTP server](#nginx-based-http-server): for ease of deployment, all HTTP configuration (TLS, listen ports, etc.) is now handled by Nginx instead of being built into Sourcegraph.
+- [Faster page load times](#faster-page-load-times): 50% smaller initial bundle size.
+
+Plus, there are several new and improved [Sourcegraph extensions](https://docs.sourcegraph.com/extensions):
+
+- [Basic code intelligence for all languages](#basic-code-intelligence-for-all-languages): fast go-to-definition and find-references for code in any language, using effective, zero-configuration heuristics when precise analysis is not configured. Enabled by default! (Also mentioned above.)
+- [Datadog metrics](#datadog-metrics-extension): easily jump to Datadog metrics information from statsd calls in your code.
+- [JavaScript/TypeScript language support](https://sourcegraph.com/extensions/sourcegraph/lang-typescript)
+- [Go language support](https://sourcegraph.com/extensions/sourcegraph/lang-go)
+- [Python language support](https://sourcegraph.com/extensions/sourcegraph/python)
+
+Also, in case you missed it, [Sourcegraph is now open source](https://about.sourcegraph.com/blog/sourcegraph-is-now-open-source) (Apache-licensed).
+
+Want to hear about new features and releases as soon as they're available? Follow [@srcgraph](https://twitter.com/srcgraph).
+
+Ready to [install or upgrade a self-hosted Sourcegraph instance](#install-or-upgrade)? You can also [try these features on Sourcegraph.com for open-source code](https://sourcegraph.com).
+
+## Highlights
+
+### Search query cheatsheet
+
+We want anyone to be able to start using Sourcegraph for code search without reading a manual. That's why we made Sourcegraph search super fast and added one-click query suggestions (computed locally using heuristics) to refine your query.
+
+But when you're searching more than a couple times daily, learning Sourcegraph's search keywords (such as `repo:`) will make you more productive. Click the new <kbd>?</kbd> icon next to any search button to see a quick reference. See ["Search query syntax"](https://docs.sourcegraph.com/user/search/queries) for more info.
+
+[Try it out](https://sourcegraph.com/search?q=repo:%5Egithub.com/kubernetes/kubernetes%24+lang:go+-f:test+-f:vendor+controller) (click the <kbd>?</kbd> next to the search button):
+
+[![Search query cheatsheet](./announcing-sourcegraph-3.0/search_query_cheatsheet.png)](https://sourcegraph.com/search?q=repo:%5Egithub.com/kubernetes/kubernetes%24+lang:go+-f:test+-f:vendor+controller)
+
+### Code intelligence for all languages
```

Comment:
@chrismwendt FYI here is how I am describing the basic code intel improvements. I think the copy could be a bit better, but it's a start.

cc @ryan-blunden